specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-01-01-preview/MigrateFromSmartDetections.json

```diff
@@ -0,0 +1,304 @@
+      "get": {
+        "tags": [
+          "MigrateFromSmartDetection"
+        ],
+        "description": "Retrieve the status of a migration process.",
+        "operationId": "MigrateFromSmartDetection_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/MigrationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MigrationStatusResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MigrationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Migration Status": {
+            "$ref": "./examples/migrateFromSmartDetectionGet.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SmartDetectionMigrationRequest": {
+      "required": [
+        "scope"
+      ],
+      "properties": {
+        "scope": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The list of resource IDs that the requested migration should be performed on."
+        },
+        "actionGroupCreationPolicy": {
+          "type": "string",
+          "enum": [
+            "Custom",
+            "Auto"
+          ],
+          "x-ms-enum": {
+            "name": "ActionGroupCreationPolicy",
+            "values": [
+              {
+                "value": "Custom",
+                "description": "Create all alert rules with a custom action group that is specified in 'customActionGroupName'."
+              },
+              {
+                "value": "Auto",
+                "description": "Automatically create an action group for each unique setting in the Smart Detection settings."
+              }
+            ],
+            "modelAsString": true
+          },
+          "description": "The policy for migrating the email settings in the Smart Detection Rules into action groups. If not specified, 'Auto' policy is used."
+        },
+        "customActionGroupName": {
+          "type": "string",
+          "description": "A custom name of an existing action group to attach to the created alert rules. Required only when actionGroupCreationPolicy is set to 'Custom'."
+        }
+      },
+      "description": "The Smart Detection migration request."
+    },
+    "MigrationStatusResponse": {
+      "description": "Response for the quota submission request.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The migration process ID.",
```

Comment:
>"description": "The migration process ID.", [](start = 10, length = 43)

This is no longer your migration process id right? It is now: "Migration resource identifier"